### ISSUE TITLE
[Server] Unexpected crash

### ISSUE THREAD

**Issue by leecalcote** [meshery/meshery#6574]
#### Current Behavior
```
cd server; cd cmd; go1.19.1 mod tidy; \
	BUILD="v0.6.29" \
	PROVIDER_BASE_URLS="https://meshery.layer5.io" \
	PORT=9081 \
	DEBUG=true \
	ADAPTER_URLS="localhost:10000 localhost:10001 localhost:10002 localhost:10004 localhost:10005 localhost:10006 localhost:10007 localhost:10009 localhost:10010 localhost:10012" \
	APP_PATH="./apps.json" \
	go1.19.1 run main.go error.go;
# command-line-arguments
ld: warning: -no_pie is deprecated when targeting new OS versions
ld: warning: non-standard -pagezero_size is deprecated when targeting macOS 13.0 or later
INFO[2022-11-24T16:12:50-06:00] Local Provider capabilities are: Not Set      app=meshery
INFO[2022-11-24T16:12:50-06:00] Meshery Server release channel is: Not Set    app=meshery
INFO[2022-11-24T16:12:50-06:00] Meshery Database is at: /Users/lee/.meshery/config  app=meshery
INFO[2022-11-24T16:12:50-06:00] Using kubeconfig at: /Users/lee/.kube         app=meshery
INFO[2022-11-24T16:12:50-06:00] Log level: debug                              app=meshery
INFO[2022-11-24T16:12:51-06:00] extracting Patterns from /Users/lee/.meshery/content/patterns  app=meshery
INFO[2022-11-24T16:12:51-06:00] seeding sample Patterns                       app=meshery
INFO[2022-11-24T16:12:51-06:00] extracting Filters from /Users/lee/.meshery/content/filters/binaries  app=meshery
INFO[2022-11-24T16:12:51-06:00] seeding sample Filters                        app=meshery
INFO[2022-11-24T16:12:51-06:00] Meshery Server listening on: 9081             app=meshery
INFO[2022-11-24T16:12:52-06:00] extracting Applications from /Users/lee/.meshery/content/applications  app=meshery
INFO[2022-11-24T16:12:52-06:00] seeding sample Applications                   app=meshery
07:31:33 http: panic serving [::1]:64643: runtime error: invalid memory address or nil pointer dereference
goroutine 16094 [running]:
net/http.(*conn).serve.func1()
	/Users/lee/sdk/go1.19.1/src/net/http/server.go:1850 +0xbf
panic({0x68b9280, 0x8ae0760})
	/Users/lee/sdk/go1.19.1/src/runtime/panic.go:890 +0x262
github.com/layer5io/meshery/server/models.(*RemoteProvider).GetSession(0xc000cd02a0?, 0x6d053c0?)
	/Users/lee/code/meshery/server/models/remote_provider.go:342 +0xa9
github.com/layer5io/meshery/server/handlers.(*Handler).validateAuth(...)
	/Users/lee/code/meshery/server/handlers/middlewares.go:91
github.com/layer5io/meshery/server/handlers.(*Handler).AuthMiddleware.func1({0x7224578, 0xc00089e000}, 0xc00046e600)
	/Users/lee/code/meshery/server/handlers/middlewares.go:68 +0x15d
net/http.HandlerFunc.ServeHTTP(0x7226c88?, {0x7224578?, 0xc00089e000?}, 0x71ee290?)
	/Users/lee/sdk/go1.19.1/src/net/http/server.go:2109 +0x2f
github.com/layer5io/meshery/server/handlers.(*Handler).ProviderMiddleware.func1({0x7224578, 0xc00089e000}, 0xc00046e300)
	/Users/lee/code/meshery/server/handlers/middlewares.go:40 +0x274
net/http.HandlerFunc.ServeHTTP(0xc00046e100?, {0x7224578?, 0xc00089e000?}, 0x800?)
	/Users/lee/sdk/go1.19.1/src/net/http/server.go:2109 +0x2f
github.com/gorilla/mux.(*Router).ServeHTTP(0xc000b42480, {0x7224578, 0xc00089e000}, 0xc00046e000)
	/Users/lee/code/go/bin/pkg/mod/github.com/gorilla/mux@v1.8.0/mux.go:210 +0x1cf
net/http.serverHandler.ServeHTTP({0xc000cd0090?}, {0x7224578, 0xc00089e000}, 0xc00046e000)
	/Users/lee/sdk/go1.19.1/src/net/http/server.go:2947 +0x30c
net/http.(*conn).serve(0xc00095c000, {0x7226c88, 0xc00165c1b0})
	/Users/lee/sdk/go1.19.1/src/net/http/server.go:1991 +0x607
created by net/http.(*Server).Serve
	/Users/lee/sdk/go1.19.1/src/net/http/server.go:3102 +0x4db
07:31:33 http: panic serving [::1]:64644: runtime error: invalid memory address or nil pointer dereference
goroutine 16096 [running]:
net/http.(*conn).serve.func1()
	/Users/lee/sdk/go1.19.1/src/net/http/server.go:1850 +0xbf
panic({0x68b9280, 0x8ae0760})
	/Users/lee/sdk/go1.19.1/src/runtime/panic.go:890 +0x262
github.com/layer5io/meshery/server/models.(*RemoteProvider).GetSession(0xc000cd0810?, 0x6d053c0?)
	/Users/lee/code/meshery/server/models/remote_provider.go:342 +0xa9
github.com/layer5io/meshery/server/handlers.(*Handler).validateAuth(...)
	/Users/lee/code/meshery/server/handlers/middlewares.go:91
github.com/layer5io/meshery/server/handlers.(*Handler).AuthMiddleware.func1({0x7224578, 0xc00089e460}, 0xc00046ea00)
	/Users/lee/code/meshery/server/handlers/middlewares.go:68 +0x15d
net/http.HandlerFunc.ServeHTTP(0x7226c88?, {0x7224578?, 0xc00089e460?}, 0x71ee290?)
	/Users/lee/sdk/go1.19.1/src/net/http/server.go:2109 +0x2f
github.com/layer5io/meshery/server/handlers.(*Handler).ProviderMiddleware.func1({0x7224578, 0xc00089e460}, 0xc00046e900)
	/Users/lee/code/meshery/server/handlers/middlewares.go:40 +0x274
net/http.HandlerFunc.ServeHTTP(0xc00046e800?, {0x7224578?, 0xc00089e460?}, 0x800?)
	/Users/lee/sdk/go1.19.1/src/net/http/server.go:2109 +0x2f
github.com/gorilla/mux.(*Router).ServeHTTP(0xc000b42480, {0x7224578, 0xc00089e460}, 0xc00046e700)
	/Users/lee/code/go/bin/pkg/mod/github.com/gorilla/mux@v1.8.0/mux.go:210 +0x1cf
net/http.serverHandler.ServeHTTP({0xc000cd0660?}, {0x7224578, 0xc00089e460}, 0xc00046e700)
	/Users/lee/sdk/go1.19.1/src/net/http/server.go:2947 +0x30c
net/http.(*conn).serve(0xc00095c140, {0x7226c88, 0xc00165c1b0})
	/Users/lee/sdk/go1.19.1/src/net/http/server.go:1991 +0x607
created by net/http.(*Server).Serve
	/Users/lee/sdk/go1.19.1/src/net/http/server.go:3102 +0x4db
07:31:33 http: panic serving [::1]:64645: runtime error: invalid memory address or nil pointer dereference
goroutine 16498 [running]:
net/http.(*conn).serve.func1()
	/Users/lee/sdk/go1.19.1/src/net/http/server.go:1850 +0xbf
panic({0x68b9280, 0x8ae0760})
	/Users/lee/sdk/go1.19.1/src/runtime/panic.go:890 +0x262
github.com/layer5io/meshery/server/models.(*RemoteProvider).GetSession(0xc000a40e40?, 0x6d053c0?)
	/Users/lee/code/meshery/server/models/remote_provider.go:342 +0xa9
github.com/layer5io/meshery/server/handlers.(*Handler).validateAuth(...)
	/Users/lee/code/meshery/server/handlers/middlewares.go:91
github.com/layer5io/meshery/server/handlers.(*Handler).AuthMiddleware.func1({0x7224578, 0xc0003fc0e0}, 0xc00040c300)
	/Users/lee/code/meshery/server/handlers/middlewares.go:68 +0x15d
net/http.HandlerFunc.ServeHTTP(0x7226c88?, {0x7224578?, 0xc0003fc0e0?}, 0x71ee290?)
	/Users/lee/sdk/go1.19.1/src/net/http/server.go:2109 +0x2f
github.com/layer5io/meshery/server/handlers.(*Handler).ProviderMiddleware.func1({0x7224578, 0xc0003fc0e0}, 0xc00040c200)
	/Users/lee/code/meshery/server/handlers/middlewares.go:40 +0x274
net/http.HandlerFunc.ServeHTTP(0xc00040c100?, {0x7224578?, 0xc0003fc0e0?}, 0x800?)
	/Users/lee/sdk/go1.19.1/src/net/http/server.go:2109 +0x2f
github.com/gorilla/mux.(*Router).ServeHTTP(0xc000b42480, {0x7224578, 0xc0003fc0e0}, 0xc00046eb00)
	/Users/lee/code/go/bin/pkg/mod/github.com/gorilla/mux@v1.8.0/mux.go:210 +0x1cf
net/http.serverHandler.ServeHTTP({0xc000cd0a80?}, {0x7224578, 0xc0003fc0e0}, 0xc00046eb00)
	/Users/lee/sdk/go1.19.1/src/net/http/server.go:2947 +0x30c
net/http.(*conn).serve(0xc00095c280, {0x7226c88, 0xc00165c1b0})
	/Users/lee/sdk/go1.19.1/src/net/http/server.go:1991 +0x607
created by net/http.(*Server).Serve
	/Users/lee/sdk/go1.19.1/src/net/http/server.go:3102 +0x4db
```

#### Environment

- **Host OS:** Mac
- **Platform:** Kubernetes
- **Meshery Server Version:** latest
- **Meshery Client Version:** none
---
#### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/build-and-release)
- 📚 [Instructions for contributing to documentation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)

**Comment by leecalcote** [comment]
@Revolyssup I'm unable to run Meshery Server.

**Comment by leecalcote** [comment]
Fixed in #6576
